liquibase-service/src/main/resources/db/rdb/routines/001-generate_data_for_data_sync_config-002.sql

```diff
@@ -0,0 +1,40 @@
+INSERT INTO [RDB].[dbo].[data_sync_config]
+(table_name, source_db, query, query_with_null_timestamp, query_count, query_with_pagination)
+VALUES
+    ('NRT_OBSERVATION', 'RDB_MODERN', 'SELECT * FROM nrt_observation WHERE last_chg_time :operator :timestamp OR refresh_datetime :operator :timestamp;', NULL, 'SELECT COUNT(*) FROM nrt_observation WHERE last_chg_time :operator :timestamp OR refresh_datetime :operator :timestamp;', 'WITH PaginatedResults AS (
```

Comment:
nitpick: do a code reformat (Intellij | Code | Reformat) so that it will automatically clean up the appearance.

Reply:
did the code reformat, but nothing change